packages/riipen-ui/src/components/TableCell.jsx

```diff
@@ -0,0 +1,65 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+class TableCell extends React.Component {
+  static propTypes = {
+    /**
+     * Children to render in the td element
+     */
+    children: PropTypes.node,
+
+    /**
+     * Classes to apply to the td element
+     */
+    classes: PropTypes.arrayOf(PropTypes.string),
+
+    /**
+     * Padding to apply to the td element
+     */
+    spacing: PropTypes.number,
```

Comment:
IMO this should be removed. All our tables should have the same spacing.

If we keep this, I think we should rename to `padding`. The UI kit's props should be as close to the CSS attribute they represent as possible when it comes to naming.

Reply:
noted